docs/DeveloperGuide.adoc

```diff
@@ -767,6 +792,229 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+
+[discrete]
+=== Use case: list tasks by date
+
+*MSS*
+
+1.  User requests to list tasks by date
+2.  AddressBook shows a list of tasks by date
+3.  User requests to see the tasks by date order
+4.  AddressBook shows the tasks by date
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The task list is invalid.
++
+[none]
+** 2a1. AddressBook shows an error message.
+** 2a2. AddressBook is empty.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: list tasks by priority order
+
+*MSS*
+
+1.  User requests to list tasks by priority
+2.  AddressBook shows a list of tasks by priority
+3.  User requests to see the tasks by priority order
+4.  AddressBook shows the tasks by priority
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The task list is invalid.
++
+[none]
+** 2a1. AddressBook shows an error message.
+** 2a2. AddressBook is empty.
++
+Use case resumes at step 2.
+
+Use case ends.
+
+[discrete]
+=== Use case: list tasks by tag
+
+*MSS*
+
+1.  User requests to list tasks by tag
+2.  AddressBook shows a list of tasks by tag
+3.  User requests to see the tasks by tag
+4.  AddressBook shows the tasks by tag
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The task list is invalid.
++
+[none]
+** 2a1. AddressBook shows an error message.
+** 2a2. AddressBook is empty.
++
+Use case resumes at step 2.
+
+Use case ends.
+
+
+[discrete]
+=== Use case: Add a new task
+
+*MSS*
+
+1.  User requests to add a task
+2.  AddressBook prompts to ask for task details
+3.  User inputs task details
+4.  AddressBook adds the task
++
+Use case ends.
+
+*Extensions*
+
+* 2a. The details is invalid.
++
+[none]
+** 2a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Edit hierarchy
+
+*MSS*
+
+1.  User requests to edit hierarchy
+2.  AddressBook prompts to ask for hierarchy details
+3.  User inputs hierarchy details
+4.  AddressBook update hierarchy
++
+Use case ends.
+
+*Extensions*
+
+* 2a. The details is invalid.
++
+[none]
+** 2a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: View task details
+
+*MSS*
+
+1.  User requests to list tasks
+2.  AddressBook shows a list of persons
+3.  User requests to see a specific task details
+4.  AddressBook shows the details of a person
```

Comment:
So is it showing a task or person? Since your project is covering both, it is important to be clear and accurate in your documentation and not get yourself and others confused.